src/main/java/com/muxiu1997/sharewhereiam/command/base/CommandWaypointBase.java

```diff
@@ -0,0 +1,71 @@
+package com.muxiu1997.sharewhereiam.command.base;
+
+import static com.muxiu1997.sharewhereiam.util.WaypointUtil.fromBase64;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import net.minecraft.client.entity.AbstractClientPlayer;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+
+import com.muxiu1997.sharewhereiam.command.CommandError;
+
+import journeymap.client.model.Waypoint;
+
+public abstract class CommandWaypointBase extends CommandBase {
+
+    final protected String commandName;
+    protected String commandUsage;
```

Comment:
Is this field ever set? Every subclass I see extends `getCoommandUsage()` instead